hyperSpec/R/palette_matlab.R

```diff
@@ -82,13 +82,43 @@ matlab.dark.palette <- function(n = 100, ...) {
 #' # Alois palette
 #'
 #' op <- par(mar = c(0, 0, 0, 0))
-#' pie(rep(1, 12), col = alois.palette(12))
+#' pie(rep(1, 12), col = palette_alois(12))
 #' par(op)
 #'
-#' plot(flu, col = alois.palette(nrow(flu)))
+#' plot(flu, col = palette_alois(nrow(flu)))
 #'
-#' plotmap(faux_cell[, , 1200], col.regions = alois.palette())
+#' plotmap(faux_cell[, , 1200], col.regions = palette_alois())
 #'
-alois.palette <- function(n = 100, ...) {
+palette_alois <- function(n = 100, ...) {
   colorRampPalette(c("black", "blue", "cyan", "green", "yellow", "red"), ...)(n)
 }
+
+# Unit tests -----------------------------------------------------------------
+#' @import hySpc.testthat
+library(hySpc.testthat)
+hySpc.testthat::test(palette_matlab_dark) <- function() {
+  context("palette_matlab_dark")
+  test_that("test palette matlab dark color", {
```

Comment:
The whole line should be as a fluent and logical sentence, e.g., 
"Test that *palette_matlab_dark() generates correct colors*"
"Test that *palette_matlab_dark() generates works*"
etc.
This sentence should describee the unit of functionality that is tested by the unit test.